terminal/Tree.cpp

```diff
@@ -155,3 +161,33 @@ void Tree::deleteRecursive(Node* entryPoint)
 	}
 }
 
+void Tree::touch(Node* node, string filename)
+{
+	if (!ifexist(node, filename))
+	{
+		Node* n = new Node();
+		head->child[head->childcount] = n;
+		head->childcount++;
+		n->parent = head;
+		n->name = filename;
+		n->runnable = true;
```

Comment:
Ez a logika majdnem ugyanaz, mint a fenti, erdemes lenne kiszedni valahova. Most meg nem latom at, miert felel a `Tree` es miert felel a `Node`. Viszont az gyanus, hogy puszta Node-okat hoztok letre, nem valami File-t, vagy ilyesmit.

Reply:
@hegyhati ezeknek a "fájloknak" amiket létrehozunk ténylegesen fájloknak kell lenni? Én arra gondoltam, hogy itt megadom a Node-nak, hogy ő egy runnable és a tartalmát is a node tartalmazná ez így nem jó akkor?

Reply:
Semmikeppen nem kell a fajlrendszeren letrehozni fajlokat. A runnable dolgot nem ertem, en csak arra utaltam, hogy van logika, ami ismetlodik, illetve hogyha ebben a kodreszletben fajlt hozunk letre, akkor miert Node-ot csinalunk, es nem File-t? Most epp csak ideneztem, nem latom at, hogy miert felel egy Node vagy egy Tree, lehet csak nem tul szerencses a nevvalaszas.

Reply:
@hegyhati igazából a névválasztás sem szerencsés, illetve a logika az lenne mögötte, hogy ha egy "Node" runnable akkor az valamilyen futtatható állomány lenne, késsőb gondoltam még felvenni neki mondjuk egy content string változót amibe kerülne a "fájl" tartalma, így végülis ténylegesen ugyan abban a rendszerben lenne megoldva a fájlok illetve a mappák kezelése.

Reply:
Hm... node nem ertem, egy file nem csak akkor file, ha futtathato. Illetve ha felveszel majd neki egy string-et, ami a content, az nem lesz jo, mert akkor annak mi lesz a tartalma a konyvtarnal? Illetve egy konytvarnak van egy "tartalma" listaja, ami meg a fajlnak nincs. A fajlra meg a konyvtarra muszaj lesz ket kulon osztalyt hasznalni, persze erdemes oket szarmaztatni egy kozos osbol.